corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media.html

```diff
@@ -1,8 +1,8 @@
 {% load i18n %}
 {% load hq_shared_tags %}
-<div id="media_image">
+<div id="{{ qualifier|default_if_none:"" }}media_image">
     <div class="control-group">
-        <label class="control-label" for="menu_image_path">{% trans "Icon" %}</label>
+        <label class="control-label" for="menu_image_path">{% trans ICON_LABEL %}</label>
```

Comment:
Is it possible to set defaults for these values (`ICON_LABEL` and `AUDIO_LABEL`)? Would something like this work:

`{% trans ICON_LABEL|default_if_none:"Icon"}`

Also what's up with ALL_CAPS (constant) names? They're template parameters, correct?